### PR TITLE
Fix local execution on OCP

### PR DESCRIPTION
**Pull request by tiraboschi**
**What this PR does / why we need it**:
When we deploy locally, HCO binary
is executed without a pod (by design) so we cannot
get a reference to its deployment and to
the owning CSV starting from the pod we are
running in.
On the other side, when deploying locally,
we know in advance the name of the deployment
object (with replica=0) and the expected
namespace from an environmental variable.
Let's get the deployment object from there.

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket
NONE
```

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: controllers/alerts/reconciler.go [hunk 1]
```diff
@@ -37,9 +38,23 @@ type MonitoringReconciler struct {
...
+	if ci.IsRunningLocally() {
+		owner = metav1.OwnerReference{
+			APIVersion:         appsv1.SchemeGroupVersion.String(),
+			Kind:               "Deployment",
+			Name:               "local",
+			UID:                uuid.NewUUID(),
+			BlockOwnerDeletion: pointer.Bool(false),
+			Controller:         pointer.Bool(false),
+		}
+		namespace = "kubevirt-hyperconverged"
```
Comment (nunnatsa) [inline controllers/alerts/reconciler.go:52]: Can we get the actual NS somehow? Not sure it will be "kubevirt-hyperconverged" in all cases.

Comment (tiraboschi) [reply controllers/alerts/reconciler.go]: done

**Comment by dharmit** [comment]
/approve

Approving since it works for me locally. Don't know if I can approve/lgtm a PR on this repo. :)

**Review by nunnatsa — APPROVED** [review]
/approve

**Pull request merged**